k8s/namespaces/mi/kustomization.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: mi
+resources:
+- namespace.yaml
+- mi-data-extractor/mi-data-extractor-identity.yaml
+- mi-data-extractor/mi-data-extractor-identity-binding.yaml
+- mi-extraction-service/mi-extraction-service-identity.yaml
+- mi-extraction-service/mi-extraction-service-identity-binding.yaml
+- mi-house-keeping-service/mi-house-keeping-service-identity.yaml
+- mi-house-keeping-service/mi-house-keeping-service-identity-binding.yaml
+- mi-integration-service/mi-integration-service-identity.yaml
+- mi-integration-service/mi-integration-service-identity-binding.yaml
+- mi-staging-service/mi-staging-service-identity.yaml
+- mi-staging-service/mi-staging-service-identity-binding.yaml
+- mi-sftp-server/deployment.yaml
```

Comment:
is this supposed to be in all environments? It looked like a test only thing

if not in all environments it should be added to the kustomize bases of the environments its required in not the mi namespace base

Reply:
the intention is to add it to all environments, I was adding it to just sandbox initially so that the MI team could have an example to follow

Reply:
specifically I mean the sftp server, it's pulling something from docker hub, is MI actually running an sftp server in kubernetes for production?